.github/workflows/build_commits.yaml

```diff
@@ -0,0 +1,42 @@
+name: build_commits
+
+
+on: [pull_request]
+
+jobs:
+  build_commits:
+    name: build commit code
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.16.3
+      - name: Cache LLVM and Clang
+        id: cache-llvm
+        uses: actions/cache@v2.1.4
+        with:
+          path: $HOME/.clang
+          key: llvm-10.0
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v1.2.1
+        with:
+          version: "10.0"
+          directory: $HOME/.clang
+          cached: ${{ steps.cache-llvm.outputs.cache-hit }}    
+      - name: Install libelf
+        run: |
+          sudo apt-get install libelf-dev
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: git fetch commit code
+        run: |
+          git fetch origin pull/$PR/head:rebase-head
+          git checkout rebase-head
```

Comment:
[Useless nit] I encourage using `git switch` over `git checkout` if the Git version allows.